.github/workflows/publish-provider-non-family.yml

```
@@ -31,24 +31,28 @@ on:
         required: false
         type: boolean
         default: true
+      registry_org:
+        description: "The organization to publish to in the Upbound Marketplace."
```

this description is out of date now that `registry_org` is used more generally
```suggestion
        description: "The organization to publish to in the targeted registries."
```